pkgs/development/ocaml-modules/tdigest/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildDunePackage
+, core
+}:
+
+buildDunePackage rec {
+  pname = "tdigest";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "SGrondin";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-R1uaCN/6NiW+jdGQiflwfihaidngvaWjJM7UFyR4vxs=";
+  };
+
+  buildInputs = [
```

Comment:
```suggestion
  propagatedBuildInputs = [
```

Reply:
I don't understand this change. As I understood it, I should try to get it to work with `buildInputs` and only use `propagatedBuildInputs` otherwise.

Reply:
Well, it does not work if `core` is not propagated, does it?

Also, please set `minimalOCamlVersion` to at least `"4.08"`. Thanks.

Reply:
That would be too easy…

Reply:
Hah. Well, I'll get there eventually. Thanks for your unfailing patience!